resource/geo-density-chart.js

```diff
@@ -84,23 +84,31 @@ function GeoDensityChart(){
         // iterate in the layers of the map
         _mapPointer.eachLayer(function(layer){
             if (layer.options.id){
```

Comment:
Select the map layer using the key layer.options.id which is only available for map